src/common/os/win32/guid.cpp

```diff
@@ -32,46 +32,23 @@
 #endif
 
 #include <windows.h>
-#include <wincrypt.h>
+#include <bcrypt.h>
 #include <objbase.h>
 #include <stdio.h>
 
 #include "firebird.h"
 #include "../common/os/guid.h"
 #include "fb_exception.h"
 
+#pragma comment(lib, "bcrypt.lib")
+
```

Comment:
So far we never uses such a way to declare dependencies.
While it is convenient for code author, it makes hard to collect all required dependencies when needed.
Thus, I prefer to use traditional way and move it into corresponding project file or property sheet.